### ISSUE TITLE
elements in a word document are not read

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#5813]
[histórico escolar anhanguera Doc1.docx](https://github.com/nvaccess/nvda/files/176520/historico.escolar.anhanguera.Doc1.docx)
Good afternoon dears.
 When trying to read a document in Word NVDA does not read the text and tables in the document.
 So that the document is read, it is necessary to search within the document an existing text in the document, so the found text gains focus, NVDA pronounces text frame and can read this text.
 To reproduce the problem, open the document in Word attached, to walk with the arrow keys or a text is read.
 Press the shortcut to locate texts, enter 100 and press enter. Now press esc, the focus returns to the document by pressing arrow down NVDA will read text frame and read the text in this table.

**Comment by jcsteh** [comment]
The text is most likely in a text box. Text boxes "float" in the 
document and can't be reached except via the mouse. We don't really have 
a good solution to this problem at the moment.

Related: #5738

**Comment by fernando-jose-silva** [comment]
thank you

**Comment by Adriani90** [comment]
yes it seems this is in a text field. I have also created an issue for this, see #8048.

**Comment by Adriani90** [comment]
Still reproducible, this is due to #6944. Closing in favor of that one.
